### DISCUSSION TITLE
which platform lavasrc use to load the track.

### DISCUSSION THREAD

**Discussion by vkamsingh** [TopiSenpai/LavaSrc#56] [Q&A] [answered]
Hello, which platform does Lavasrc use to load tracks for Spotify, Dezzer, or Apple Music? because, as far as I know, the Spotify API does not support streams tracks.

**Comment by topi314** [comment, marked as answer]
as stated in the `README.md` spotify & apple music are loaded from whatever providers you configure & the rest are natively played from their sources

**Reply by topi314** [reply]
@vkamsingh this is the place to reply to an answer, that's how discussions work

**Reply by topi314** [reply]
see https://github.com/TopiSenpai/LavaSrc#configuration
